sklearn/lda.py

```diff
@@ -1,6 +1,20 @@
 import warnings
+from .discriminant_analysis import LinearDiscriminantAnalysis as _LDA
```

Comment:
Okay. I thought it would still be good to not make the `LinearDiscriminantAnalysis` class available at `sklearn.lda`. Do you feel this change should be reverted?

If yes, apologies @waterponey for the additional work. I'll buy you a beer if we meet :p